cas-server-core-api-authentication/src/main/java/org/jasig/cas/authentication/AuthenticationSystemSupport.java

```diff
@@ -35,6 +35,17 @@
     /**
      * Initiate potential multi-transaction authentication event by handling the initial authentication transaction.
      *
+     * @param authentication a pre-established authentication object in a multi-legged authentication flow.
+     *
+     * @return authentication result builder used to accumulate authentication transactions in this authentication event.
+     *
+     * @since 4.3.0
+     */
+    AuthenticationResultBuilder handleInitialAuthenticationTransaction(Authentication authentication);
```

Comment:
On naming things.. There we go ;-) I think this particular API method is not about "handling" the init tx, but more about an establishment of the multi-legged authentication context from an already handled initial transaction (elsewhere). So I'd rename it to `AuthenticationResultBuilder establishAuthenticationContextFrom(Authentication)` or `AuthenticationResultBuilder fromInitial(Authentication)`